### PR TITLE
Add Work and About Me sections

### PR DESCRIPTION
**Pull request by smunoz1988**
-Add work sections in differents cards
-Add about me section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,5 +38,115 @@ <h4 class="connect">LET'S CONNECT</h4>
...
+                <ul class="languaje">
+                    <li class="js"><img src="images/js.png"><p>JavaScript</p></li>
+                    <li class="html"><img src="images/html.png"><p>HTML</p></li>
+                    <li class="css"><img src="images/css.png"><p>CSS</p></li>
+                </ul>
+                <li class="atribute">Frameworks</li>
+                <hr>
+                <li class="atribute">Skills</li>
+                <hr>
+            </ul>
```
Comment (abdallahmalima) [inline index.html]: - [ ] You have done a great job 👍so far in making your project look like the Figma design. Please🙏 add the missing elements that are found in the Figma design, which will make your project look like the original Figma design. This is a good indicator that the section is clickable.
image

**Review by abdallahmalima — CHANGES_REQUESTED** [review]
Hi @smunoz1988 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Align content with display grind ✔
- Align content with display flex ✔
- Follow HTLM/CSS best practices ✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -38,5 +38,123 @@ <h4 class="connect">LET'S CONNECT</h4>
...
+                </ul>
+                <div class="container">
+                    <p class="atribute">Frameworks</p>
+                    <img class="arrow" src="images/arrowR.png">
+                </div>
+                <hr>
+                <div class="container">
+                    <p class="atribute">Skills</p>
+                    <img class="arrow" src="images/arrowr.png">
+                </div>
```
Comment (souad988) [inline index.html]: Well done, only one small thing fix the typo in the link :

arrow typo

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @smunoz1988 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Only one step to the approval !


## Highlights
✔Proffesional README.
✔Implemented almost all requirements .

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- adding a margin between about me section and languages 

**your project**
Capturemargin student

**figma design**
margin


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @smunoz1988,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- Nice Ui Design.
- Professional README.
- Very Pixel Perfect

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**